crates/valence_new/src/client.rs

```diff
@@ -64,6 +65,8 @@ pub struct Client {
     entities_to_despawn: Vec<VarInt>,
     got_keepalive: bool,
     last_keepalive_id: u64,
+    keepalive_sent_time: Instant,
+    ping: u32,
```

Comment:
Might be good to use an `i32` here and default to `-1` since this is what the player list expects.